LibraryManagement/MemberForm.xaml

```diff
@@ -60,23 +60,32 @@
         <!--#endregion-->
 
         <!--#region Image attachment-->
-        <Ellipse
-            Fill="#cccccc"
+        <Image
```

Comment:
I Dont think this is how it should be.

Reply:
ellipse was given for strokes

Reply:
🕳  Keeping this on Hold! - until working with images.